app/helpers/infra_networking_helper/textual_summary.rb

```diff
@@ -0,0 +1,37 @@
+module InfraNetworkingHelper::TextualSummary
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    %i(switch_type)
+  end
+
+  def textual_group_relationships
+    %i(hosts)
+  end
+
+  def textual_group_smart_management
+    %i(tags)
+  end
+
+    #
+  # Items
+  #
+
+  def textual_switch_type
+    {:label => _("%{switch} Type") % {:switch=> ui_lookup(:table => "switches")}, :value => @record.shared}
```

Comment:
I do have the function here  - but it is not used on the screen - I will remove it.